standard/standard/annex-glossary.adoc

```diff
@@ -1,97 +1,231 @@
-[appendix]
-:appendix-caption: Annex 
+[[annex-glossary]]
+[appendix,obligation=informative]
 == Glossary
```

Comment:
@ronaldtse, according to the [documentation,](https://www.metanorma.org/author/ogc/authoring-guide/terms-definitions/#terminology-clauses) this attribute: `[heading='terms and definitions']`, should be added.
Though it seems that it doesn't make any different to the current result.

Reply:
Maybe the keyword “Glossary” is now supported officially? Ping @opoudjis   if so we should fix the documentation. Thanks!

Reply:
@manuelfuenmayor no, this is incorrect. According to documentation, we do not need an additional `heading`:
<img width="776" alt="Screen Shot 2022-08-15 at 1 05 10 PM" src="https://user-images.githubusercontent.com/11865/184579485-5d7cda6a-4eef-42f1-b4f1-46fc054b91a0.png">

Just `[appendix]` and `== Glossary` is sufficient.